workers/data_refinery_workers/downloaders/geo.py

```diff
@@ -138,11 +145,11 @@ def download_geo(job_id: int) -> None:
 
     file_assocs = DownloaderJobOriginalFileAssociation.objects.filter(downloader_job=job)
 
-    original_file = file_assocs[0].original_file 
+    original_file = file_assocs[0].original_file
```

Comment:
If these are unique now you can use `.get`

Reply:
Mmm I didn't change any of the logic around this, just deleted a trailing whitespace. It does seem like they should be unique, however before I changed it I guess I'd just be curious if there was any reason you didn't do it like that in the first place.